build.gradle.kts

```
@@ -44,3 +44,6 @@ kover {
         }
     }
 }
+tasks.dokkaHtmlMultiModule {
+    pluginsMapConfiguration.set(mapOf("org.jetbrains.dokka.base.DokkaBase" to """{ "templatesDir" : "${projectDir.toString().replace('\\', '/')}/dokka-templates" }"""))
```

```suggestion
    pluginsMapConfiguration.set(mapOf("org.jetbrains.dokka.base.DokkaBase" to """{ "templatesDir" : "${projectDir.resolve("dokka-templates")}" }"""))
```
Or there's a reason to manually replace backslashes?